tests/unit/style/parsing/image.rs

```diff
@@ -10,6 +10,22 @@ use style::values::specified::image::*;
 use url::Url;
 
 #[test]
+fn test_linear_gradient() {
+    // Parsing from the right
+    assert_roundtrip_with_context!(Image::parse, "linear-gradient(to left, red, green)",
+                                                   "linear-gradient(4.712389rad, red, green)");
```

Comment:
It's hard to resolve that kind a nits with code I think. It can be implemented but it'll give so much false negatives.